### PR TITLE
Add hostAliases for webserver

### PR DESCRIPTION
**Pull request by ignacionf**
The hostAliases option for webservice is added since it was only available for workers

### CODE REVIEW

**Review by potiuk — APPROVED** [review]
LGTM. @jedcunningham @dstandish ?

File: chart/Chart.lock [hunk 1]
```diff
@@ -1,5 +1,6 @@
 dependencies:
 - name: postgresql
+  repository: ""
```
Comment (jedcunningham) [inline chart/Chart.lock:3]: Can you make this change in a separate PR?

**Comment by bdsoha** [comment]
@ignacionf Is this still relevant, something similar was already merged #27544?

**Comment by potiuk** [comment]
Yep. Closing for now - we can re-open if the author wants to change it.
